frontend/src/app/main/ui/ds/spacing.scss

```diff
@@ -8,6 +8,7 @@
 
 $_sp-2: px2rem(2);
 $_sp-4: px2rem(4);
+$_sp-6: px2rem(8);
```

Comment:
why do you add px2rem(8) when you call the variable `_sp-6` I thing this is a typo